sphinx_needs/css/dark/data_tables.css

```diff
@@ -0,0 +1,101 @@
+/* DataTable layout*/
```

Comment:
I would like to also remove this, but there wasn't obvious duplication here, so not as trivial